src/test/java/com/timgroup/statsd/BlockingStatsDClientTest.java

```diff
@@ -0,0 +1,11 @@
+package com.timgroup.statsd;
+
+/**
+ * @author sfreitag
+ */
+public class BlockingStatsDClientTest extends StatsDClientTest<BlockingStatsDClient> {
+    @Override
+    public BlockingStatsDClient buildClient(String prefix, String[] constantTags) {
+        return new BlockingStatsDClient(prefix, STATSD_SERVER_HOST, STATSD_SERVER_PORT, constantTags);
+    }
+}
```

Comment:
If anything, we should have more tests in the `NonBlocking`, since that is more sophisticated. I feel pretty comfortable with the existing tests covering `Blocking`.